docs/content/guides/dagster/managing-ml.mdx

```diff
@@ -0,0 +1,266 @@
+
+### Automating ML model maintenance
+
+Whether you have a large or small model, Dagster can help automate data refreshes and model training based on your business needs.
+
+We can use [auto-materializing assets](/concepts/assets/asset-auto-execution) to update a machine learning model when the upstream data is updated. This can be done by setting the `AutoMaterializePolicy` to `eager`, which means that our machine learning model asset will be refreshed anytime our data asset is updated.
+
+We can also prevent the model from being refreshed too often by using the `max_materilizations_per_minute` setting, which is recommended to have more control over how often the model is retrained. This approach can be suitable for smaller machine learning models where the cost of training is low, and the value of having the complete data used to train a machine learning model is essential.
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=eager_materilization_start endbefore=eager_materilization_end
+from dagster import AutoMaterializePolicy, asset
+
+
+@asset
+def my_data():
+    ...
+
+
+@asset(
+    auto_materialize_policy=AutoMaterializePolicy.eager(
+        max_materializations_per_minute=60
+    )
+)
+def my_ml_model(my_data):
+    ...
+```
+
+Some machine learning models might more cumbersome to retrain; it also might not be as important to update them as soon as new data arrives. For this, we can use a lazy auto-materialization policy which can be used in two different ways. The first, by using it with a `freshness_policy` as shown below. In this case, my_ml_model will only be auto-materialized once a week.
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=lazy_materlization_start endbefore=lazy_materlization_end
+from dagster import AutoMaterializePolicy, asset, FreshnessPolicy
+
+
+@asset
+def my_other_data():
+    ...
+
+
+@asset(
+    auto_materialize_policy=AutoMaterializePolicy.lazy(),
+    freshness_policy=FreshnessPolicy(maximum_lag_minutes=7 * 24 * 60),
+)
+def my_other_ml_model(my_other_data):
+    ...
+```
+
+his can be useful if you know that you want your machine learning model retrained at least once a week. While Dagster allows you to refresh a machine learning model as often as you like, best practice is to re-train as seldomly as possible. Model retraining can be costly to compute and having a minimal number of model versions can reduce the complexity of reproducing results at a later point in time.
```

Comment:
```suggestion
This can be useful if you know that you want your machine learning model retrained at least once a week. While Dagster allows you to refresh a machine learning model as often as you like, best practice is to re-train as seldomly as possible. Model retraining can be costly to compute and having a minimal number of model versions can reduce the complexity of reproducing results at a later point in time.
```

Fixing a typo